util/etcd/election_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package etcd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/coreos/etcd/clientv3"
+	"github.com/google/trillian/testonly/integration/etcd"
+)
+
+var (
+	// client is an etcd client. Initialized by TestMain().
+	client *clientv3.Client
+)
+
+func TestMain(m *testing.M) {
+	_, c, cleanup, err := etcd.StartEtcd()
```

Comment:
Might be nicer to do this in the test itself, cf. `integration/quota/quota_test.go`  `TestEtcdRateLimiting`